src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4878,30 +4849,23 @@ private TypeParameterSyntax ParseTypeParameter()
                     this.AddError(CreateMissingIdentifierToken(), ErrorCode.ERR_IdentifierExpected));
             }
 
-            var attrs = _pool.Allocate<AttributeListSyntax>();
-            try
+            var attrs = default(SyntaxList<AttributeListSyntax>);
```

Comment:
Is there any risk that users who expected to get an empty attribute list will now get a 'default' list?

Reply:
no.  as far as `SyntaxList<T>` is concerned, they're the same thing.  i.e. `default` is *how* syntaxlist represents an empty list.

Reply:
@RikkiGibson ^